releasenotes/notes/31573.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: EnvoyFilter
+issue:
+  - 31573
+releaseNotes:
+  - |
+    **Fixed** a bug causing HTTP_FILTER don't support INSERT_FIRST for EnvoyFilter
```

Comment:
Maybe something like: `**Fixed** a bug where the EnvoyFilter HTTP_FILTER didn't support INSERT_FIRST`

Reply:
Thanks, I will fix it